packages/core/src/components/Lightbox/LightboxImage.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
```

Comment:
`Image.tsx`

Reply:
👍 I'll need to use the name `LightboxImage` in the Lightbox itself to differentiate from the plain javascript `Image` component

Reply:
You can rename on import: `import LightboxImage from './Image';`. The file itself can be `Image.tsx`.